src/platform/site-wide/sass/modules/_m-homepage-blog.scss

```diff
@@ -0,0 +1,27 @@
+.homepage-blog {
+  &__image {
+    height: 100%;
+
+    img {
+      width: 100%;
+      height: 350px;
+      object-fit: cover;
+      object-position: top;
+      display: block;
+
+      @media (min-width: $small-screen) {
+        height: 100%;
+      }
+    }
+  }
+
+  .va-h-ruled--white {
+    border-bottom: 1px solid $color-white;
+    padding: 1rem 0 1rem 0;
```

Comment:
Same comment, but for padding: https://design.va.gov/foundation/utilities/padding